internal/cmd/beta/security-group/create/create.go

```diff
@@ -0,0 +1,197 @@
+package create
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/goccy/go-yaml"
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/client"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	Labels      map[string]any
```

Comment:
changed datatype to *map[string]any

Reply:
Any reason to prefer `map[string]any` instead of `map[string]string` like in the other commands?

Reply:
?I have changed the datatype, please take a look [here](here: https://github.com/stackitcloud/stackit-cli/blob/8fef4991d0daa640a2628b22c070c18b4c61e2b3/internal/cmd/beta/security-group/create/create.go#L30). Or am I missing something here?

Reply:
The problem was that the generated model used a map[string]any. So I used the same type for the input model to avoid another conversion when the request is build. I can change this of course, but what would the preferred way here?

Reply:
I didn't notice before that the flag type for the `labels` is wrong currently. It should be a `cmd.Flags().StringToString` flag (instead of a `cmd.Flags().StringSlice`). This already does the validation you are doing manually in `parseInput`, and would simplify that to a one liner of `Labels:          flags.FlagToStringToStringPointer(p, cmd, labelFlag)`

Reply:
See e.g. here: https://github.com/stackitcloud/stackit-cli/blob/c715b0ac904913e15db4550b8710f2910b6e39a3/internal/cmd/beta/server/create/create.go#L177

And here: https://github.com/stackitcloud/stackit-cli/blob/c715b0ac904913e15db4550b8710f2910b6e39a3/internal/cmd/beta/server/create/create.go#L260